backend/dal/Migrations/v1.0.14.1/Up/PostUp/01_PIMS_CONTACT_VW.sql

```diff
@@ -0,0 +1,44 @@
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop view dbo.PIMS_CONTACT_VW
+PRINT N'Drop view dbo.PIMS_CONTACT_VW'
+GO
+DROP VIEW IF EXISTS [dbo].[PIMS_CONTACT_VW]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+PRINT N'Create view dbo.PIMS_CONTACT_VW'
+GO
+CREATE VIEW [dbo].[PIMS_CONTACT_VW] AS
+SELECT CONCAT('P', PERSON_ID) AS ID, PERSON_ID, NULL AS ORGANIZATION_ID, p.IS_DISABLED, TRIM(CONCAT_WS(' ',FIRST_NAME, MIDDLE_NAMES, SURNAME)) AS SUMMARY, SURNAME, FIRST_NAME, MIDDLE_NAMES, NULL AS ORGANIZATION_NAME, p.ADDRESS_ID, a.STREET_ADDRESS_1 as MAILING_ADDRESS, a.MUNICIPALITY_NAME, ps.DESCRIPTION AS PROVINCE_STATE FROM PIMS_PERSON p
```

Comment:
In order for a view in ef core to be the principle in during property navigation, there must be a unique key. Since this table is a union of persons and organizations, just use the person id and organization id (made unique by appending P or O).